src/AppBundle/Controller/Admin/BlogController.php

```diff
@@ -93,6 +96,10 @@ public function newAction(Request $request)
             // See http://symfony.com/doc/current/book/controller.html#flash-messages
             $this->addFlash('success', 'post.created_successfully');
 
+            if ($form['createAndShow']->isClicked()) {
```

Comment:
@javiereguiluz,  the `Form` class implements the `ArrayAccess` interface, internally it uses the `get` method, so the `$form['createAndShow']` is equal to the `$form->get('createAndShow')`, but more short. http://github.com/symfony/form/blob/master/Form.php#L949